lib/elixir/src/elixir_errors.erl

```diff
@@ -374,7 +374,10 @@ parse_error(Location, File, Error, <<"[", _/binary>> = Full, Input) when is_bina
 %% Given a string prefix and suffix to insert the token inside the error message rather than append it
 parse_error(Location, File, {ErrorPrefix, ErrorSuffix}, Token, Input) when is_binary(ErrorPrefix), is_binary(ErrorSuffix), is_binary(Token) ->
   Message = <<ErrorPrefix/binary, Token/binary, ErrorSuffix/binary >>,
-  raise_snippet(Location, File, Input, 'Elixir.SyntaxError', Message);
+  case pop_key(Location, error_type) of
```

Comment:
You can use lists:keytake :)

Reply:
well, I guess they could have named it keypop :laughing: